themes/bootstrap3/js/channels.js

```diff
@@ -86,8 +96,6 @@ VuFind.register('channels', function Channels() {
     $(op).on('swipe', function channelDrag() {
       switchPopover(false);
     });
-    // truncate long titles and add hover
```

Comment:
Should the "add hover" comment still be retained here?

Reply:
Add hover was out of date before this change

Reply:
Ahh, right, the popover thing is not a hover. :-)